lib/jemoji.rb

```diff
@@ -3,9 +3,12 @@
 require 'html/pipeline'
 
 module Jekyll
-  class Emoji < Jekyll::Generator
+  class Emoji < Jekyll::Converter
     attr_reader :config
     safe true
+    priority :low
+
+    EMOJIABLE_EXTS = %w{.html .htm .md .markdown .mkdn .mkd .textile}
 
```

Comment:
It's really anything whose output ext is HTML. But this should only operate on HTML input, really, but the `matches` works for input ext only.